### PR TITLE
Add some issue templates

### PR DESCRIPTION
**Pull request by astoycos**
Add some issue templates to which is being completed as part of https://github.com/kubernetes-sigs/network-policy-api/pull/119

### CODE REVIEW

**Comment by astoycos** [comment]
PTAL @npinaeva

File: .github/ISSUE_TEMPLATE/enhancement-proposal.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Enhancement Proposal
+about: Use this template for submitting enhancement requests
+title: "[ENHANCEMENT]"
+labels: ''
```
Comment (npinaeva) [inline .github/ISSUE_TEMPLATE/enhancement-proposal.md:5]: do we want a label for better filtering like `kind/enhancement` or `kind/feature`?

Comment (astoycos) [reply .github/ISSUE_TEMPLATE/enhancement-proposal.md:5]: Sure I'll add one!

**Comment by npinaeva** [comment]
/lgtm
I will update my PR, and when this one merges we can test the link :)

**Pull request merged**